### PR TITLE
:boom: take url by prefix

### PR DESCRIPTION
**Pull request by vikiival**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [ ] Closes #<issue_number>
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 35058de</samp>

Refactored `utils/chain.ts` to simplify and fix the chain endpoint utilities.

### <samp>🤖 Generated by Copilot at 35058de</samp>

> _To improve the chain endpoint tools_
> _The coder used maps and fixed rules_
> _He removed an import_
> _That was not in support_
> _And avoided some type error fools_

### CODE REVIEW

File: utils/chain.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { ENDPOINTS } from '@kodadot1/vuex-options'
+import { ENDPOINT_MAP } from '@kodadot1/static'
 import { chainList } from '@kodadot1/static'
```
Comment (Jarsen136) [inline utils/chain.ts:3]:
```suggestion
import { ENDPOINT_MAP, chainList } from '@kodadot1/static'
```

File: utils/chain.ts [hunk 2]
```diff
@@ -97,7 +96,7 @@ export const disableChainListOnBetaEnv = [
   'snek',
 ]
 
-export const availablePrefixes = (): Option[] => {
+export const availablePrefixes = (): any[] => {
```
Comment (Jarsen136) [inline utils/chain.ts:99]:
```suggestion
export const availablePrefixes = (): Option[] => {
```

**Comment by vikiival** [comment]
Thanks for the review @Jarsen136 ❤️

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Pull request merged**